probe/host/host.go

```diff
@@ -145,12 +156,14 @@ func (s *Server) CheckThreshold(info Info) (bool, string) {
 		}
 		message += "Memory Shortage!"
 	}
-	if s.Threshold.Disk > 0 && s.Threshold.Disk <= info.Disk.Usage/100 {
-		status = false
-		if message != "" {
-			message += " | "
+	for _, disk := range info.Disks {
+		if s.Threshold.Disk > 0 && s.Threshold.Disk <= disk.Usage/100 {
+			status = false
+			if message != "" {
+				message += " | "
+			}
+			message += fmt.Sprintf("Disk Full! - [%s]", disk.Tag)
```

Comment:
```suggestion
			message += fmt.Sprintf("Disk usage exceeds threshold! - [%s]", disk.Tag)
```

Reply:
but the threshold does not necessarily mean that the disk is full :(
I believe that since we define our own thresholds these messages need to be adapted to state that. 
As an example my threshold is at 70% so when i pass the threshold my disks are not Full.

Reply:
I mean we should give the same pattern for all of the resource usages.

And considering the thresholds we would use to determine the health(up/down) of the host, so, it's better to give the related message. 

I agree the "Full" might be too serious, maybe we can say: Disk Space Low!

Reply:
> I mean we should give the same pattern for all of the resource usages.
Yes i agree with that, i'd like the messages to be consistent also. 

Lets ignore this particular suggestion so as to proceed with the main PR which is to add support for disks. We can address these messages in another PR to make sure the messages are consistent and more appropriate for each case?

Reply:
no problem